pipeline/image.py

```diff
@@ -0,0 +1,56 @@
+from collections import defaultdict
+import os
+import datajoint as dj 
+from skimage import io, util
+
+schema = dj.schema(dj.config['custom']['database.prefix'] + 'image')
+
+# === Useful functions===
+def get_image_paths():
+    paths = dj.config.get('custom', {}).get('image_root_data_dir',None)
+    return paths
+
+
+@schema
+class ImageInfo(dj.Manual):
+    definition = """
+    #Image information
+    image_id   : varchar(24)  #name of image
+    ---
+    filepath   : varchar(50)  #image location relative to image_root_data_dir
+    """
+
+
+@schema
+class Image(dj.Imported):
+    definition = """
+    #Image
+    -> ImageInfo
+    ---
+    image  : longblob   #the image itself
+    """
+
+    def make(self,key):
+        #to do join filepath and 
+        dirname = get_image_paths()
+        filepath = (ImageInfo & key).fetch1('filepath')
+        full_file_path = os.path.join(dirname, filepath)
```

Comment:
I'd recommend using the `pathlib` standard library. Use can import from that the `Path` class. It provides an object-oriented way to create paths that can be used cross-platform. Extremely powerful and once you get used to it, you won't want to go back!

Reply:
We will be using pathlib.